### ISSUE TITLE
Colors of messages should depend on contact

### ISSUE THREAD

**Issue by stefan-niedermann** [nerzhul/ocsms#148]
# Current behavior

- **Header bar** gets a random color
- **Received msg** is grey
- **Sent msg** is green

# Expected behavior

- **Header bar** gets a random color
- **Received msg** should get a color based on the Header bar color
- **Sent msg** should be grey

# Example

- **Header bar** is `rgb(124, 116, 231)` (purple)
- **Received msg border** should be `rgba(124, 116, 231, .6)` (purple)
- **Received msg background** should be `rgba(124, 116, 231, .15)` (a semi transparent version of the purple that is used in the Header bar)
- **Sent msg** should be like todays Received msg - grey

Pay attention to the *alpha-channel* usage: border = `.6`, background = `.15`

# Benefit

1. Less colors, less cluttering
2. clear assignment of messages to the contact
3. Less important stuff (sent msg) is inconspicuos

**Comment by nerzhul** [comment]
interesting idea, can you provide a PR ?

**Comment by stefan-niedermann** [comment]
Yeah, but depends on how much time i can invest... i will maybe need some weeks to set up development environement.

**Comment by nerzhul** [comment]
no problem, we have time :) i'm on many libre projects not time to code everything together

**Comment by jayhybrid** [comment]
Wasn't this fixed here https://github.com/nextcloud/ocsms/issues/149? I am also in favour of consistent colouring. @stefan-niedermann are you still available for this?

**Comment by stefan-niedermann** [comment]
Unfortunately not, i am currently very busy with [Nextcloud Deck](https://github.com/stefan-niedermann/nextcloud-deck) and [Nextcloud Notes](https://github.com/stefan-niedermann/nextcloud-notes)...
